heralding/capabilities/http.py

```diff
@@ -65,11 +65,27 @@ def do_GET(self):
             headers_bytes = bytes(self.headers['Authorization'], 'utf-8')
             self.wfile.write(headers_bytes)
             self.wfile.write(b'not authenticated')
-
-    # Disable logging provided by BaseHTTPServer
+            aux_data = self.get_auxiliary_info()
+            self._session.add_auxiliary_info(aux_data)
+            # Disable logging provided by BaseHTTPServer
     def log_message(self, format_, *args):
         pass
 
+    def get_auxiliary_info(self):
+          data_fields = HTTPHandler.get_aux_fields()
+          data_list = {}
+          for i in data_fields:
+              data_list.update({str(i) : str(self.headers[str(i)])})
```

Comment:
What happens if “i” does not exist in the headers dict?

Reply:
Please, use some other name for variable here instead of `i`. For example `f` or `field`.

Reply:
> What happens if “i” does not exist in the headers dict?

Well, we are choosing what headers we want to log in get_aux_fields() but in the event we enter something wrong we can simply check whether that header exists or not. If it exists, we can add the relevant data else we can create a column heading of passed header and mark it's value as "Header Value Not Found.".

> Please, use some other name for variable here instead of i. For example f or field.

 Sure thing.

I have made changes regarding this, please see if that's what we would like!